config/sets/symfony/symfony53.php

```diff
@@ -24,7 +24,7 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->sets([SymfonySetList::ANNOTATIONS_TO_ATTRIBUTES]);
 
-    $rectorConfig->import(__DIR__ . '/symfony53-types.php');
+    $rectorConfig->import(__DIR__ . '/symfony50-types.php');
```

Comment:
I am not entirely sure if this should be even included as part of this set?

Reply:
Not at all, it should be removed.

I'm surprised PHPStan missed this.